e2e/helpers/environment/GhostManager.ts

```diff
@@ -28,8 +29,8 @@ export class GhostManager {
     }
 
     /** High-level: create, wait, and return a GhostInstance description. */
-    async startInstance(instanceId: string, siteUuid: string): Promise<GhostInstance> {
-        const container = await this.createAndStart({instanceId, siteUuid});
+    async startInstance(instanceId: string, siteUuid: string, portalUrl?: string): Promise<GhostInstance> {
+        const container = await this.createAndStart({instanceId, siteUuid, portalUrl});
```

Comment:
```suggestion
    async startInstance(instanceId: string, siteUuid: string, config: GhostConfig): Promise<GhostInstance> {
        const container = await this.createAndStart({instanceId, siteUuid, configl});
```

non-blocking: I think we should generalize `startInstance` to accept an arbitrary configuration object here instead of listing them out individually. This config object would then get passed to Ghost on boot via environment variable. This way we can pass in labs flags or arbitrary config (via the test fixture) to the Ghost instance we are starting.

As an added bonus, we can slowly start building out at least a typescript type to document Ghost's configuration.